firstuseauthenticator/templates/reset.html

```diff
@@ -0,0 +1,38 @@
+{% extends "page.html" %}
+
+{% block main %}
+
+<div class="container">
+<form action="{{post_url}}" method="post" role="form">
+  <h2 class="auth-form-header">
+    Change Password
+  </h2>
+  <div class='auth-form-body'>
+
+
+    <label for='password_input'>New Password:</label>
+    <input
+      type="password"
+      class="form-control"
+      name="password"
+      id="password_input"
+      tabindex="2"
+    />
+
+    <input
+      type="submit"
+      id="login_submit"
+      class='btn btn-jupyter'
+      value='Sign In'
+      tabindex="3"
+    />
+  </div>
+</form>
+{% if result %}
```

Comment:
Can you render this using a 'success' alert, like https://getbootstrap.com/docs/3.3/components/#alerts? That would make it more obvious I think.